TheWitcher.Tests/HeroServiceTests.cs

```diff
@@ -216,5 +217,154 @@ public void BuyWeaponsTestByNullAndNegativeParameters(int heroId, int clothesId,
             //assert
             Assert.Equal(resultOfTestExpected, resultOfTest);
         }
+        [Fact]
+        public void CheckHeroDTOItemByNull()
+        {
+            //arrange
+            var mockUnitOfWork = new Mock<IUnitOfWork>();
```

Comment:
Please, move Mock objects to class, make them fields and initialize them inside a constructor (or testStartUp). Then just setup them directly inside a method